core/src/main/java/io/fabric8/maven/core/service/openshift/ImageStreamService.java

```diff
@@ -230,4 +238,19 @@ private String findTagSha(OpenShiftClient client, String imageStreamName, String
             throw new MojoExecutionException("Could not find a tag in the ImageStream " + imageStreamName);
         }
     }
+
+    private boolean isDate1AfterDate2(String date1, String date2) {
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
```

Comment:
Hmm, the reason why it was giving false all the time was due to the date format we're using in `SimpleDateFormat`; it should be `yyyy-MM-dd'T'HH:mm:ss'Z'`